aspnetcore/security/authentication/community.md

```diff
@@ -0,0 +1,19 @@
+# Community OSS Authentication options
+
+When developing ASP.NET Core applications you can use [ASP.NET Core Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity) to secure your web applications. You can also replace that with another authentication provider. This page contains community provided open source authentication options for ASP.NET Core. This page will be periodically updated as new providers become available.
+
+
+This page contains community provided open source authentication options for ASP.NET Core.
+
+
+
+# OSS Authentication Providers
+
+The list below is sorted alphabetically.
+
+| Name                      | Description | Link |
+|:--------------------------|------------:|------------------------------------------------:|
+| Identity Server 4 | Open Source OpenID Connect and OAuth 2.0 framework for .NET | https://identityserver.io/ |
```

Comment:
Please use this for IdentityServer4:

"OpenID Connect and OAuth 2.0 framework for ASP.NET Core - officially certified by the OpenID Foundation and under governance of the .NET Foundation"